Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:16.04 
+USER root
+
+WORKDIR /tests
+
+ARG STAGE_CONFIG="{'zut':'allures'}"
+
+ENV MOZ_HEADLESS="1"
+ENV RUST_BACKTRACE="full"
+
+RUN apt-get -y update && \
+    apt-get install -y software-properties-common &&\
+    apt-get install -y wget unzip && \
+    apt-get install -y libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev  && \
+    cd /usr/src
+
+RUN apt-get install -y libgtk-3-0 libdbus-glib-1-2 libgtk-3-dev libglib2.0-dev
+
+#TODO: do we need all of these or just one?
+RUN rm -rf /var/lib/apt/lists/* &&  \
+    apt-get -y --purge dist-upgrade && \
+    apt-get -y --purge autoremove -qq && \
+    apt-get clean -y
+
+#TODO: confirm if we need all moz-central or just TPS test dir?
+RUN mkdir -p /mnt/extra/mozilla-central/testing/tps && \
+    wget -q https://github.com/mozilla/gecko-dev/archive/master.zip 
+
+# testing/tps
+RUN unzip master.zip 'gecko-dev-master/testing/tps/*' -d /tmp && \
+    mkdir -p /mnt/extra/mozilla-central/testing && \
+    mv /tmp/gecko-dev-master/testing/tps /mnt/extra/mozilla-central/testing 
+
+# services/sync
+RUN unzip master.zip 'gecko-dev-master/services/sync/*' -d /tmp && \
+    mkdir -p /mnt/extra/mozilla-central/services && \
+    mv /tmp/gecko-dev-master/services/sync /mnt/extra/mozilla-central/services 
+
+# testing/modules
+RUN unzip master.zip 'gecko-dev-master/testing/modules/*' -d /tmp && \
+    mv /tmp/gecko-dev-master/testing/modules /mnt/extra/mozilla-central/testing 
+
+# cleanup
+RUN rm -rf /tmp && \
+    rm master.zip
+
+RUN wget -O firefox-nightly.tar.bz2 'https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US' && \
+    tar xjf firefox-nightly.tar.bz2 && \
+    mv -i firefox firefox-nightly && \
+    rm firefox-nightly.tar.bz2
+
+#TODO: confirm, do we need virtualenv at all?
+RUN cd /mnt/extra/mozilla-central/testing/tps && \
+    ./create_venv.py /tests/tps-stage 
+
+CMD . /tests/tps-stage/bin/activate && \
+    echo ${STAGE_CONFIG} > stage-config.json &&\
+    /tests/tps-stage/bin/runtps --debug --binary="/tests/firefox-nightly/firefox" --configfile=/tests/stage-config.json
```

Comment:
need to parameterize "tps-stage" to accomodate variable test ENV (stage or prod)